### PR TITLE
Create Login Component

### LINKED ISSUE #7: [Frontend] Create Login component
# Description

Refer the ui design and create the Login component

# Screenshot

image

# Colours

image
image
image
 ### Login button : Gradient - 5731BA and 291757

# link

For reference you can see the design prototype : 
https://www.figma.com/proto/dMw9GAeJja2baVhDoceAEB/Untitled?node-id=9%3A48&viewport=-1544%2C148%2C0.4358828365802765&scaling=contain

# illustrations

image

### PR DESCRIPTION
**Pull request by piall**
### Description

I have created a Login component. Let me know if have anything to fix.  :smiley: 
### Related Issue

Fixes #7   [Frontend] Create Login component

### Type of Change:

- Code
- User Interface

### Checklist:

- [x] Pull request is made against `development` branch.
- [x] My code follows the style guidelines of this project.
- [x] I run `npm run format` before creating the pull request.
- [x] I have performed a self-review of my own code.
- [x] My changes generate no new warnings

- [x] UI made replicates the design provided

### CODE REVIEW

**Comment by Abhishek-kumar09** [comment]
@mdPial Have you linked the component somewhere, I mean I could only see the Login Component file, and no changes in any other file where it could be linked or called from.

You can read about how to show your content in popup and dialogs here: https://material-ui.com/components/dialogs/

**Comment by piall** [comment]
@Abhishek-kumar09 No I have not linked in my PR. In the issue I asked about it and got answer to just create the component for now.
image

**Comment by Abhishek-kumar09** [comment]
Okay, okay no problem. Unless you don't link it to some existing components or create separate route How can I view your changes.

Sorry the comment was bit misunderstood by shitiz

If you are getting some problem in linking and how to show components you can ping me anytime

**Comment by piall** [comment]
It's okay. I was also thinking the same thing how will you see the changes then thought might be there is a cool way to do that😁😁

**Comment by piall** [comment]
I am having issue on this
image
I couldn't add custom `width and height` in **px** in dialog. Could you help me on this? I wanted to add custom dialog width and height so that full components can be visible and background color to none so that UI look like same.

**Comment by Abhishek-kumar09** [comment]
Push the code and will see

**Comment by piall** [comment]
I have updated last time I had forgotten to add separate closing option :sweat_smile:

**Comment by piall** [comment]
I don't know why checks are not successful. It's running without any problem in my localhost.

**Comment by Abhishek-kumar09** [comment]
image
No unused variables should be there. image should have an alt prop.

> I don't know why checks are not successful. It's running without any problem in my localhost.

**Comment by piall** [comment]
Updated. Is there any way to check those locally?

**Comment by Abhishek-kumar09** [comment]
> Updated. Is there any way to check those locally?

Ya these errors also pops up in the console

File: src/components/Login.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        </Typography>
+        <Typography className={classes.details}>
+          Welcome back,
+          <br />
+          please login to your account
+        </Typography>
+        <Typography>Email Id</Typography>
+        <input className={classes.input} variant="outlined" />
+        <Typography>Password</Typography>
+        <input className={classes.input} variant="outlined" />
```
Comment (Abhishek-kumar09) [inline src/components/Login.js]:
```diff
-        <input className={classes.input} variant="outlined" />
-        <Typography>Password</Typography>
-        <input className={classes.input} variant="outlined" />
+        <input className={classes.input} type="email" variant="outlined" />
+        <Typography>Password</Typography>
+        <input className={classes.input} type="password" variant="outlined" />
```

File: src/layouts/MainLayout/TopBar/Account.js [hunk 2]
```diff
@@ -16,9 +15,8 @@ import {
   CircularProgress,
   makeStyles
 } from '@material-ui/core';
-import authService from 'src/services/authService';
-import StyledFirebaseAuth from 'react-firebaseui/StyledFirebaseAuth';
 import { login, dismissLogin, logout } from 'src/actions/accountActions';
+import Login from 'src/components/Login';
 
 const useStyles = makeStyles(theme => ({
   button: {
```
Comment (Abhishek-kumar09) [inline src/layouts/MainLayout/TopBar/Account.js:22]: To fixup the border-radius thing:

add one more class:
```
  dialog: {
    '& .MuiDialog-paper': {
      borderRadius: '20px'
    }
  }
```

and put that class here :
```diff
        <Dialog
          maxWidth
          open={!user && !!loginFlag}
          onClose={handleClose}
          aria-labelledby="form-dialog-title"
+          className={classes.dialog}
        >
          <Login handleClose={handleClose} />
        </Dialog>
```

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
Some minors

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
Changes are superawesome for desktop, On mobile:
image

Put the image taking whole width(xs={12})
then put the login components inputs and buttons

**Comment by mdPial** [comment]
I am facing difficulty to make Card responsive. Could you guide me by resources?

**Comment by piall** [comment]
I have updated

**Comment by Abhishek-kumar09** [comment]
Thanks @mdPial Loved it!

**Comment by Abhishek-kumar09** [comment]
You can add yourself in contributor list in a new PR, and could start grabbing other issues

**Comment by piall** [comment]
It's my pleasure @Abhishek-kumar09 . Thank you for helping.

**Pull request merged**